src/cascadia/TerminalSettingsModel/ActionArgs.idl

```diff
@@ -168,6 +188,11 @@ namespace Microsoft.Terminal.Settings.Model
         UInt64 ContentId{ get; set; };
 
         String ToCommandline();
+
+        UInt32 GetArgCount();

```

Comment:
It might be better to put these four methods in their own interface (`IActionArgsDescriptorAccess`), and inherit that interface in IActionArgs _and_ NewTerminalArgs.

Right now, they _look_ the same but they cannot be used in the same way.

If it is literally the same `interface`, you can unify any codepaths that try to access `INewTerminalArgs.GetArgCount` and `IActionArgs.GetArgCount` (et al).

The true benefit of IDL is that you can define interfaces that encapsulate common behavior, after all. :)